spec/kong_tests.conf

```diff
@@ -21,8 +21,6 @@ anonymous_reports = off
 
 dns_hostsfile = spec/fixtures/hosts
 
-lua_package_path = ?/init.lua;./kong/?.lua
```

Comment:
We don't need this one (never really needed it) because we rely on the default value provided by `kong_defaults.lua`